travis-ci build dependency error

2.55s$ git clone --depth=50 --branch=develop https://github.com/metricly/heapster.git metricly/heapster
Cloning into 'metricly/heapster'...
remote: Counting objects: 8738, done.
remote: Compressing objects: 100% (5336/5336), done.
remote: Total 8738 (delta 3650), reused 7203 (delta 2879), pack-reused 0
Receiving objects: 100% (8738/8738), 9.45 MiB | 12.89 MiB/s, done.
Resolving deltas: 100% (3650/3650), done.
$ cd metricly/heapster
$ git checkout -qf c212a69240c1cf9ba997dc3e497100c95e576760
8.95s$ GIMME_OUTPUT="$(gimme 1.8 | tee -a $HOME/.bashrc)" && eval "$GIMME_OUTPUT"
go version go1.8 linux/amd64
$ export GOPATH=$HOME/gopath
$ export PATH=$HOME/gopath/bin:$PATH
$ mkdir -p $HOME/gopath/src/github.com/metricly/heapster
$ rsync -az ${TRAVIS_BUILD_DIR}/ $HOME/gopath/src/github.com/metricly/heapster/
$ export TRAVIS_BUILD_DIR=$HOME/gopath/src/github.com/metricly/heapster
$ cd $HOME/gopath/src/github.com/metricly/heapster
0.01s
$ gimme version
v1.5.0
$ go version
go version go1.8 linux/amd64
go.env
$ go env
GOARCH="amd64"
GOBIN=""
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/travis/gopath"
GORACE=""
GOROOT="/home/travis/.gimme/versions/go1.8.linux.amd64"
GOTOOLDIR="/home/travis/.gimme/versions/go1.8.linux.amd64/pkg/tool/linux_amd64"
GCCGO="gccgo"
CC="gcc"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build580633227=/tmp/go-build -gno-record-gcc-switches"
CXX="g++"
CGO_ENABLED="1"
PKG_CONFIG="pkg-config"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
Using Go 1.5 Vendoring, not checking for Godeps
install
0.00s$ true
3.95s$ make metricly
rm -f heapster
rm -f eventer
hooks/check_boilerplate.sh
hooks/check_gofmt.sh
hooks/run_vet.sh
can't load package: package github.com/metricly/heapster/metrics/apis/metrics: code in directory /home/travis/gopath/src/github.com/metricly/heapster/metrics/apis/metrics expects import "k8s.io/heapster/metrics/apis/metrics"
can't load package: package github.com/metricly/heapster/metrics/apis/metrics/v1alpha1: code in directory /home/travis/gopath/src/github.com/metricly/heapster/metrics/apis/metrics/v1alpha1 expects import "k8s.io/heapster/metrics/apis/metrics/v1alpha1"
find . -type f -name "*.go" | grep -v "./vendor*" | xargs gofmt -s -w
GOARCH=amd64 CGO_ENABLED=0 go build -ldflags "-w -X k8s.io/heapster/version.HeapsterVersion=v1.5.0-beta.3 -X k8s.io/heapster/version.GitCommit=c212a69" -o heapster k8s.io/heapster/metrics
can't load package: package k8s.io/heapster/metrics: cannot find package "k8s.io/heapster/metrics" in any of:
	/home/travis/.gimme/versions/go1.8.linux.amd64/src/k8s.io/heapster/metrics (from $GOROOT)
	/home/travis/gopath/src/k8s.io/heapster/metrics (from $GOPATH)
make: *** [build] Error 1

--- a/.travis.yml
+++ b/.travis.yml
@@ -11,7 +11,7 @@ go:
   - 1.8
 
 before_install:
-  - go get -d -t github.com/metricly/go-client
+  - go get -d github.com/metricly/go-client
 
 script:
   - make metricly

--- a/.travis.yml
+++ b/.travis.yml
@@ -11,7 +11,7 @@ go:
   - 1.8
 
 before_install:
-  - go get -t github.com/metricly/go-client
+  - go get -d -t github.com/metricly/go-client
 
 script:
   - make metricly

--- a/.travis.yml
+++ b/.travis.yml
@@ -11,7 +11,8 @@ go:
   - 1.8
 
 before_install:
-  - go get -d github.com/metricly/go-client
+  - go get -d github.com/metricly/go-client/api
+  - go get -d github.com/metricly/go-client/model/core
 
 script:
   - make metricly

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,6 +2,8 @@ sudo: required
 
 language: go
 
+go_import_path: k8s.io/heapster
+
 services:
   - docker
 

--- a/.travis.yml
+++ b/.travis.yml
@@ -10,6 +10,9 @@ services:
 go:
   - 1.8
 
+before_install:
+  - go get -t github.com/metricly/go-client
+
 script:
   - make metricly
 

